index.html

```diff
@@ -200,5 +200,64 @@ <h4>Skills</h4>
       </ul>
     </div>
   </section>
+  <section id="contact-form" class="flex-column">
+    <p>I'm always <br>interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</p>
+    <form action="https://formspree.io/f/xzbykgyg" method="POST">
+      <ul class="flex-column">
+        <li id="name-label">
+          <label>
+            <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" 
+            required>
+          </label>
+        </li>
+        <li>
+          <hr>
+          <label>
+            <input type="email" id="email" name="email" placeholder="Email address" required>
+          </label>
+        </li>
+        <li>
+          <hr>
+          <textarea name="message" id="message" cols="31" rows="5" 
+          placeholder="Enter text here" maxlength="500" required></textarea>
+        </li>
+        <li>
+          <button id="form-button" type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Can you please add the `cursor: pointer;` to the button to give a good user experience?